tests/core/files/opds2/pq_auth_token_feed.json

```diff
@@ -0,0 +1,158 @@
+{
+    "metadata": {
+        "title": "Ebook Central OPDS Feed for columbia",
+        "itemsPerPage": 2,
+        "currentPage": 1,
+        "numberOfItems": 2
+    },
+    "links": [
+        {
+            "href": "https://ebookcentral.proquest.com/auth/ws/auth/PlainPartnerAuthToken/columbia?userName={patron_id}",
```

Comment:
Might be good to do a find / replace on this file, and make the links a bit more generic. Maybe just replace `ebookcentral.proquest.com` with `example.com`, so we don't end up accidentally hitting the PQ servers in our tests. Maybe replace `lib/columbia` with `test` as well. And drop `pq_` from the filename.